source/install/setting-up-local-machine-using-docker.rst

```diff
@@ -34,101 +34,97 @@ If you have Docker set up, Mattermost installs in one line:
 
-.. code:: bash
+      .. code:: bash
       
-   sudo dnf -y install dnf-plugins-core
-   sudo dnf config-manager \
-   --add-repo \
-   https://download.docker.com/linux/fedora/docker-ce.repo
-   sudo dnf install docker-ce docker-compose git # Accepting the new docker repository key
-   sudo usermod -aG docker <username>
-   sudo systemctl start docker
+         sudo dnf -y install dnf-plugins-core
+         sudo dnf config-manager \
+         --add-repo \
+         https://download.docker.com/linux/fedora/docker-ce.repo
+         sudo dnf install docker-ce docker-compose git # Accepting the new docker repository key
+         sudo usermod -aG docker <username>
+         sudo systemctl start docker
  
-2. Start Docker container: 
+    2. Start Docker container: 
 
-.. code:: bash
+      .. code:: bash
       
-   docker run --name mattermost-preview -d --publish 8065:8065 --add-host dockerhost:127.0.0.1 mattermost/mattermost-preview
+         docker run --name mattermost-preview -d --publish 8065:8065 --add-host dockerhost:127.0.0.1 mattermost/mattermost-preview
        
-3. When Docker is done fetching the image, open http://localhost:8065/ in your browser.
-
-Arch Linux
-~~~~~~~~~~
-
-To install the preview on Arch Linux, see the `installation guide <https://wiki.archlinux.org/index.php/Mattermost#With_Docker>`__ on the Arch Linux wiki.
+    3. When Docker is done fetching the image, open http://localhost:8065/ in your browser.
+  
+  .. tab:: Arch Linux
+  
+    To install the preview on Arch Linux, see the `installation steps <https://wiki.archlinux.org/index.php/Mattermost#With_Docker>`__ on the Arch Linux wiki.
 
 Setting up SMTP email (Recommended) 
 -----------------------------------
 
-The default single-container Docker instance for Mattermost is designed for product evaluation, and sets ``SendEmailNotifications=false`` so the product can function without enabling email. To see the product's full functionality, enabling SMTP email is recommended.
-
-.. include:: ../configure/smtp-email.rst 
-  :start-after: How to Enable Email
+The default single-container Docker instance for Mattermost is designed for product evaluation, and sets ``SendEmailNotifications=false`` so the product can function without enabling email. To see the product's full functionality, we recommend enabling SMTP email. See the `SMTP Email Setup <https://docs.mattermost.com/configure/smtp-email.html>`__ documentation for details.
 
 Configuration Settings
 ----------------------
 
-See `Configuration Settings <https://docs.mattermost.com/configure/configuration-settings.html#gitlab-settings>`__ documentation to customize your deployment.
+See the `Configuration Settings <https://docs.mattermost.com/configure/configuration-settings.html>`__ documentation to customize your deployment.
 
 Updating Docker Preview
```

Comment:
Should this be about updating or deleting? The commands below are stopping and deleting the container.

Reply:
Good catch. I've updated this section for clarity. Please let me know if additional changes are needed.